app/src/main/java/com/ncnf/feed/ui/FeedFragment.java

```diff
@@ -103,14 +103,14 @@ protected void onEventClick(SocialObject e) {
 
         ConstraintLayout feedContainer = globalWindow.findViewById(R.id.feed_event_container);
         FrameLayout feedFrame = globalWindow.findViewById(R.id.feed_event_fragment);
-        Button feedButton = globalWindow.findViewById(R.id.feed_event_button);
+        //Button feedButton = globalWindow.findViewById(R.id.feed_event_button);
```

Comment:
If we do not use the button anymore etc, we should delete it instead of leaving it invisible.

Reply:
I didn't delete it yet because there is still a small issue with the back button in one particular place so I thought it'd be better to keep the code stashed somewhere in case we decide to go back to using it.